packages/boiler-task-selenium/src/runner/nightwatch/runner.js

```diff
@@ -0,0 +1,11 @@
+const nightwatch = require('nightwatch');
+
+const argv = JSON.parse(process.argv[2]);
+
+nightwatch.runner(argv, function(success) {
+  if (!success) {
+    process.exit(1);
+  } else {
+    process.exit(0);
+  }
```

Comment:
should probably put a comment in here jus so we remember we have to run this as a child process or all hell breaks loose, also can't run with `babel-node`